test/stdlib/TestJSONEncoder.swift

```diff
@@ -409,6 +409,199 @@ class TestJSONEncoder : TestJSONEncoderSuper {
                    nonConformingFloatDecodingStrategy: decodingStrategy)
   }
 
+  // MARK: - Key Strategy Tests
+  private struct EncodeMe : Encodable {
```

Comment:
Helper types are defined in this file below all of the tests, if you feel like following that convention (not really necessary, just consistent).